docs/roi_circle.md

```diff
@@ -24,7 +24,7 @@ from plantcv import plantcv as pcv
 # or "plot" (Jupyter Notebooks or X11)
 pcv.params.debug = "print"
```

Comment:
Should we consider changing these to "plot" I think that would be more beginner friendly.